cmd/aws-iam-authenticator/init.go

```diff
@@ -73,6 +74,10 @@ var initCmd = &cobra.Command{
 }
 
 func init() {
+	viper.AutomaticEnv()
+	viper.SetEnvPrefix("aws_iam_authenticator")
```

Comment:
i think this will make https://github.com/kubernetes-sigs/aws-iam-authenticator/blob/master/cmd/aws-iam-authenticator/root.go#L68 not to work. can we not have the automatic env and use bind for the ones we need ?

Reply:
I'll test if `KUBERNETES_AWS_AUTHENTICATOR_CLUSTER_ID` doesn't work anymore.
As I understand it, automatic env doesn't affect paramters with hard coded env var name.

Reply:
i think automatic env overrides the specific one https://github.com/spf13/viper/blob/1508a7ba4478e59a385419023ed91eeeb8d522b4/viper.go#L1207-L1218